src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/WebCmdletElementCollection.cs

```diff
@@ -18,7 +18,7 @@ internal WebCmdletElementCollection(IList<PSObject> list)
         }
 
         /// <summary>
-        /// Finds the element with name or id
+        /// Finds the element with name or id.
         /// </summary>
         /// <param name="nameOrId"></param>
         /// <returns></returns>
```

Comment:
suggest populating or removing the empty <returns></returns> comment blocks for these properties.

Reply:
Fixed.